src/components/backend-ai-multi-select.ts

```diff
@@ -0,0 +1,262 @@
+        }
+
+        mwc-button {
+          margin: var(--selected-item-margin, 3px);
+          --mdc-theme-primary: var(--selected-item-theme-color);
+          --mdc-theme-on-primary: var(--selected-item-theme-font-color);
+          --mdc-typography-font-family: var(--selected-item-font-family);
+          --mdc-typography-button-font-size: var(--selected-item-font-size);
+          --mdc-typography-button-text-transform: var(--selected-item-text-transform);
+        }
+
+        mwc-button[unelevated] {
+          --mdc-theme-primary: var(--selected-item-unelevated-theme-color);
+          --mdc-theme-on-primary: var(--selected-item-unelevated-theme-font-color);
+        }
+
+        mwc-button[outlined] {
+          --mdc-theme-primary: var(--selected-item-outlined-theme-color);
+          --mdc-theme-on-primary: var(--selected-item-outlined-theme-font-color);
+        }
+        
+        mwc-list {
+          font-family: var(--general-font-family);
+          width: 100%;
+          position: absolute;
+          left: 0;
+          right: 0;
+          z-index: 1;
+          border-radius: var(--select-background-border-radius);
+          background-color: var(--select-background-color, #efefef);
+          --mdc-theme-primary: var(--select-primary-theme);
+          --mdc-theme-secondary: var(--select-secondary-theme);
+          box-shadow: var(--select-box-shadow);
+        }
+
+        mwc-list > mwc-check-list-item {
+          background-color: var(--select-background-color, #efefef);
+        }
+
+        .selected-area {
+          background-color: var(--select-background-color, #efefef);
+          border-radius: var(--selected-area-border-radius, 5px);
+          border: var(--selected-area-border, 1px solid rgba(0,0,0,1));
+          padding: var(--selected-area-padding, 10px);
+          min-height: var(--selected-area-min-height, 24px);
+          height: var(--selected-area-height, auto);
+        }
+
+        .expand {
+          transform:rotateX(180deg) !important;
+        }
+
+      `
+    ];
+  }
+
+  /**
+   * Show Dropdown Menu
+   */
+  _showMenu() {
+    this._modifyListPosition(this.items.length);
+    this.shadowRoot.querySelector('#menu').style.display = '';
+  }
+
+  /**
+   * Hide Dropdown Mneu
+   */
+  _hideMenu() {
+    const dropdownIcon = this.shadowRoot.querySelector('#dropdown-icon');
+    dropdownIcon.on = false;
+    dropdownIcon.classList.remove('expand');
+    this.shadowRoot.querySelector('#menu').style.display = 'none';
+  }
+
+  /**
+   * Toggle visibility of dropdown menu
+   * 
+   * @param {Event} e - Click from expand/shrink icon in right-side of selected-area
+   */
+  _toggleMenuVisibility(e) {
+    const dropdownIcon = this.shadowRoot.querySelector('#dropdown-icon');
+    dropdownIcon.classList.toggle('expand');
+    if (e.detail.isOn) {
+      this._showMenu();
+    } else {
+      this._hideMenu();
+    }
+  }
+
+  /**
+   * Set the position of Dropdown menu
```

Comment:
Can you add an explanation/description of why we need to modify the position of the list item using this method for future reference?